---
title: Comparative single-cell transcriptional atlases of Babesia species reveal conserved and species-specific expression profiles
date: 2022-09-28
categories:
  - Computational Biology
  - Parasitology
  - Single-cell Genomics
authors:
  - yrezvani
  - ckeroack
  - belsworth
  - aarriojas
  - mjgubbels
  - mtduraisingh
  - kzarringhalam
journal: PLOS Biology, 20(9), e3001816 (2022)
journal_url: https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001816
pubmed: 36137068
doi: 10.1371/journal.pbio.3001816
template: publication-post.html
---

## Abstract

Babesia is a genus of apicomplexan parasites that infect red blood cells in vertebrate hosts. Pathology occurs during rapid replication cycles in the asexual blood stage of infection. Current knowledge of Babesia replication cycle progression and regulation is limited and relies mostly on comparative studies with related parasites. Due to limitations in synchronizing Babesia parasites, fine-scale time-course transcriptomic resources are not readily available. Single-cell transcriptomics provides a powerful unbiased alternative for profiling asynchronous cell populations. Here, we applied single-cell RNA sequencing to 3 Babesia species (B. divergens, B. bovis, and B. bigemina). We used analytical approaches and algorithms to map the replication cycle and construct pseudo-synchronized time-course gene expression profiles. We identify clusters of co-expressed genes showing "just-in-time" expression profiles, with gradually cascading peaks throughout asexual development. Moreover, clustering analysis of reconstructed gene curves reveals coordinated timing of peak expression in epigenetic markers and transcription factors. Using a regularized Gaussian graphical model, we reconstructed co-expression networks and identified conserved and species-specific nodes. Motif analysis of a co-expression interactome of AP2 transcription factors identified specific motifs previously reported to play a role in DNA replication in Plasmodium species. Finally, we present an interactive web application to visualize and interactively explore the datasets.

<!-- more -->

## Key Points

- First single-cell transcriptional atlases of *Babesia* species
- Identification of distinct developmental stages and their molecular signatures
- Comparative analysis reveals conserved and species-specific expression patterns
- Novel computational approaches for parasite scRNA-seq analysis
- Interactive web-based resources for exploring *Babesia* transcriptomes
- Integration with bulk transcriptomic and genomic data enhances interpretability

## Supporting Materials

- [**Babesia Cell Atlas**](https://umbibio.math.umb.edu/babesiasc)
- [**Analysis Code**](https://github.com/umbibio/scBabesiaAtlases)
- [**Analysis Code (Zenodo)**](https://doi.org/10.5281/zenodo.7036500)
- [**Atlas Dashboard Code**](https://github.com/umbibio/babesia_dash_app)
- [**Data**](https://umbibio.math.umb.edu/data/babesia/)

## Citation

```bibtex
@article{rezvani2022comparative,
  title={Comparative single-cell transcriptional atlases of Babesia species reveal conserved and species-specific expression profiles},
  author={Rezvani, Yasaman and Keroack, Caroline D and Elsworth, Brendan and Arriojas, Argenis and Gubbels, Marc-Jan and Duraisingh, Manoj T and Zarringhalam, Kourosh},
  journal={PLOS Biology},
  volume={20},
  number={9},
  pages={e3001816},
  year={2022},
  publisher={Public Library of Science},
  doi={10.1371/journal.pbio.3001816},
  pmid={36137068},
  pmcid={PMC9531838}
}
```
